internal/controllers/machineset/machineset_controller.go

```diff
@@ -518,8 +528,8 @@ func shouldExcludeMachine(machineSet *clusterv1.MachineSet, machine *clusterv1.M
 	return false
 }
 
-// adoptOrphan sets the MachineSet as a controller OwnerReference to the Machine.
-func (r *Reconciler) adoptOrphan(ctx context.Context, machineSet *clusterv1.MachineSet, machine *clusterv1.Machine) error {
+// updateControllerReference sets the MachineSet as a controller OwnerReference to the Machine.
+func (r *Reconciler) updateControllerReference(ctx context.Context, machineSet *clusterv1.MachineSet, machine *clusterv1.Machine) error {
```

Comment:
nit: let's be consistent between MD and MS and pick either updateControllerRef or updateControllerReference